### DISCUSSION TITLE
How can I install veros offline?

### DISCUSSION THREAD

**Discussion by sylenola** [team-ocean/veros#644] [Q&A] [answered]
My computer is offline.

I download the source code elsewhere and unzip it on my computer. Then I get veros-main file-folder under route: D:\soft\veros-main\veros-main.

I then run 'cd D:\soft\veros-main\veros-main' and 'pip install -e ./veros' in powershell, error occured:
![PXL_20240827_065501012](https://github.com/user-attachments/assets/36897cfb-d437-4d0f-bea5-795f2527e248)

**Comment by dionhaefner** [comment, marked as answer]
I'm guessing you need to use `pip install -e .` instead if you're already in the veros folder. Not that it's going to do you much good though, since the dependencies are downloaded during install.

I guess you could do `pip download veros`, then copy the downloaded wheels to an offline machine, and use `pip install --no-index --find-links . veros` to install if you really need to do this.

**Reply by sylenola** [reply]
Thank you!
